### ISSUE TITLE
add ability to list running UD dashboards

### ISSUE THREAD

**Issue by Cragdoo** [adamdriscoll/poshprotools#96]
something along the lines of `list-UDDashboard`

Results

Name                   Port   
-------                  ------

Dashboard1        9999
Dashboard2        7777

would it be possible to also name the process when it's running so it would look like 

Name                   Port   
-------                  ------

ServerDash        9999
CloudReport      7777

**Comment by adamdriscoll** [comment]
There is Get-UDDashboard. Does that work? 

It will only list dashboards running in the current process. Do you want a cmdlet to list all running dashboards on a server?

**Comment by Cragdoo** [comment]
yeah Get-UDDashboard works, don't know why I didn't just try that at first. But yes to your second question :)

**Comment by adamdriscoll** [comment]
Ok. Can you rename this Issue to something related to working cross-process and I can put on my todo list?

**Comment by Cragdoo** [comment]
done

**Comment by adamdriscoll** [comment]
thanks

**Comment by Cragdoo** [comment]
just discovered the -Name option, to allow the dashboard to be called whatever you like :)
